apis/opts/networks.go

```diff
@@ -94,7 +95,8 @@ func ValidateNetworks(nwConfig *types.NetworkingConfig) error {
 		for k := range nwConfig.EndpointsConfig {
 			l = append(l, k)
 		}
-
+		// make sure l is sorted
+		sort.Strings(l)
```

Comment:
why we sort this....

Reply:
the same questions to previous ulimit testing ...